test/ScanForErrorMiddleware/ScannerTest.php

```diff
@@ -125,10 +139,13 @@ public function testScanningEmitsInfoToConsoleWhenEncounteringFilesOfInterest()
         $this->console
             ->writeLine(
                 Argument::that(function ($arg) {
-                    if (! strstr($arg, 'src/ErrorMiddleware.php')) {
+                    if (false === strpos($arg, 'src/ErrorMiddleware.php')) {
```

Comment:
The original was correct; `strstr` returns boolean `false` if the needle is not found in the haystack. The only reason it might emit a false positive is if it matches exactly at the beginning, as it would return `''` in that case; in this particular comparison, that would never happen...